ts/dateManipulation.ts

```diff
@@ -1,25 +1,35 @@
 export const WEEKDAYS = 7;
 export const HOUR_COUNT = 24;
-export const LOCALE = "us-US";
 export const MILISECOND_HOUR = 60 * 60 * 1000;
 
-export const getToday = () => new Date(Date.now());
+export type localeType = "us-US" | "lt-LT";
```

Comment:
You can also use `enum` as a type. With this, you will be able to write something like....
```
enum localeTypes {
    US: 'us-US',
    LT: 'lt-LT'
}

let locale = localeTypes.US;
```

No seemingly "magic" strings.